### PR TITLE
Feature/wish list item and options insert or update

### PR DESCRIPTION
**Pull request by tessguefen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,4 +67,65 @@ Any of the 3 category identifiers can be used: Category_Id, Category_Code, Edit_
...
+		}
+	]
+}
+```
+
+**Example Resonse**
+
+```json
+{
+    "success": 1,
```
Comment (pdeans) [inline README.md]: Remove the comma

File: README.md [hunk 2]
```diff
@@ -67,4 +67,65 @@ Any of the 3 category identifiers can be used: Category_Id, Category_Code, Edit_
...
+| --- | --- | --- |
+| Module_Function | String | WishListItemAndOptions_InsertOrUpdate |
+| Customer_Login | String | Customer Login to use. |
+| Product_Code | String | Product code to use. |
+| Quantity | Number | Quantity of product you wish to add. |
+| Wishlist_ID | Number | _Optional_ Wishlist ID. |
+| Attributes | Array | Optional, unless the product requires attributes. |
+
+```
+Attributes is an array with the following parameters for each attribute:
```
Comment (pdeans) [inline README.md]: This should say something like Attributes is an array of objects with the following properties for each attribute object:

Comment (pdeans) [reply README.md]: Should also specify the Name, Type, and Description for the Attribute object.

Comment (tessguefen) [reply README.md]: No you

Comment (pdeans) [reply README.md]: Are we able to add fields for wish list notes, date/time an item was added, and/or if the list is public or private?

Comment (tessguefen) [reply README.md]: Are you talking about the Wishlist or the items
Date/time no, that is hardcoded in miva.
Wishlist Item notes, yes i can make a spot for that.

For wishlist, not the items, that would need to be a seperate function

Comment (pdeans) [reply README.md]: wishlist - title / public vs private
wishlist item - notes

File: README.md (hunk 1 above)
Comment (tessguefen) [reply README.md]: This has been fixed.

File: README.md (hunk 2 above)
Comment (tessguefen) [reply README.md]: Now that the function is changing, i will need to change the docs. So i'm going to ignore that for now and make the updates when necessary.

**Comment by tessguefen** [comment]
@pdeans please review changes.

**Pull request merged**